### ISSUE TITLE
MPV Options "hwdec=yes"

### ISSUE THREAD

**Issue by hugok79** [celluloid-player/celluloid#610]
Using Celluloid with MPV Extra command "hwdec=yes" Linux Mint 20.1 Cinnamon, freezes Celluloid 0.20

**Comment by gnome-mpv** [comment]
Please attach a [log file](https://github.com/celluloid-player/celluloid/blob/master/.github/CONTRIBUTING.md).

**Comment by hugok79** [comment]
I would like to say that this occurs with **hwdec=yes**, with **hwdec=no** or without any option.

I'm testing linux mint 20.1 cinnamon beta because i am the Linux Mint(as well as Celluloid) Portuguese(pt) translator without any changes, in live mode and i just open a .mp4 movie... although Celluloid plays the movie, clicking on the close button or anywhere else it takes too much time to react. If clicking on the close button takes a long time until Celluloid closes.

[Ceeluloid_log.zip](https://github.com/celluloid-player/celluloid/files/5703087/Ceeluloid_log.zip)

**Comment by gnome-mpv** [comment]
>I would like to say that this occurs with hwdec=yes, with hwdec=no or without any option.

Seems like this isn't related to `hwdec` then.

>although Celluloid plays the movie, clicking on the close button or anywhere else it takes too much time to react

Are you using Intel iGPU? I think this might be related to the Intel iGPU performance regression that happened around Linux 5.7. On my Haswell machine, I'm also getting freezes when using more demanding options like `scale=ewa_lanczos` even though they worked perfectly fine with older kernels.

**Comment by hugok79** [comment]
Yes, i'm testing in a old laptop, (Acer Extensa 5620Z), it must be because of that,
